src/gui/application.cpp

```diff
@@ -640,12 +643,13 @@ void Application::showHelp()
 
     stream << QLatin1String("File synchronisation desktop utility.") << endl
            << endl
-           << QLatin1String(optionsC);
+           << optionsC();
 
-    if (_theme->appName() == QLatin1String("ownCloud"))
+    if (_theme->appName() == QLatin1String("ownCloud")) {
```

Comment:
This is the second time I read more or less this exact same comparison in different parts of the code.
Should we maybe have something like theme->isBranded() and/or theme->isUnbranded()? 

Totally not needed for this PR obviously...

Reply:
Thought the same when I did the last change....
But we have also a check for testpilotcloud